### DISCUSSION TITLE
Ookii.Dialogs.Wpf 5.0.0 dropped support for .NET 4.6.1

### DISCUSSION THREAD

**Discussion by dittodhole** [ookii-dialogs/ookii-dialogs-wpf#75] [General]
Hi there,

is there any particular reason for dropping .NET 4.6.1? I am wondering, as I see .NET 4.6.2 support (I also inspected the [official migration guide](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/runtime/4.6.1-4.6.2) and found no blocker ... or am I missing something?) 🤔 

Any chance to bring support for .NET 4.6.1 back?

Best
Andreas

**Comment by augustoproiete** [comment]
Hey Andreas, thanks for the question!

There's no _strong_ technical reason to drop .NET Framework 4.6.1 _right now_, but my thoughts were the following:

1. I've added support for .NET 6, which requires Visual Studio 2022
   - Visual Studio 2022 does not install .NET Framework 4.6.1 by default, so anyone building Ookii Dialogs from source would need to install the .NET Framework 4.6.1 Targeting Pack separately
2. Windows Update [installs new versions of the .NET Framework automatically](https://devblogs.microsoft.com/dotnet/category/dotnet-framework/) for most users, so it's likely that most users have .NET Framework 4.8 installed already, or at least .NET  Framework 4.7.2, so dropping 4.6.1 in favor of 4.6.2 didn't seem like it would make much difference
3. .NET Framework [4.6.1 will reach end of life on April 2022](https://devblogs.microsoft.com/dotnet/net-framework-4-5-2-4-6-4-6-1-will-reach-end-of-support-on-april-26-2022/), so only a few months left anyway
4. Ookii Dialogs v4.0.0 will continue to work, and it supports .NET 4.6.1

I can be convinced to add .NET Framework 4.6.1 back if there's enough demand, but it would only be for a few more months anyway... The next release after April 2022 would drop it again.

I'd be interested in hearing about cases where users only have .NET Framework 4.6.1 on their machines and reasons they can't upgrade do 4.6.2

**Comment by dittodhole** [comment]
Thanks for the clarification!

> I'd be interested in hearing about cases where users only have .NET Framework 4.6.1 on their machines and reasons they can't upgrade do 4.6.2

I do have a strong industrial background in my day job, where we can't force our customers (with customized WSUS policy) to upgrade (think of the EOL of Windows XP; our customers are still undergoing the upgrade ...) 🤔
